### PR TITLE
creating a file for Chennai, india

### LINKED ISSUE #12: Add a hometown file for Mayakrishnan
# Mayakrishnan's Hometown

Mayakrishnan would like to introduce to the class, and share more details.

@halfaro1 Please let me know what to do?

### PR DESCRIPTION
**Pull request by mayakrishnan**
Resolves #12 
- [x] Created a branch
- [x] Added a file
- [x] Committed the file
- [x] Opened a Pull request
- [x] Have a discussion
- [x] Make more commits
- [x] Merge pull request

### CODE REVIEW

File: places/##mayakrishnan-hometown.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#Chennai, India
+## Historic place
```
Comment (mayakrishnan) [inline places/##mayakrishnan-hometown.md:2]: Can you provide more details of historic places please?

Comment (mayakrishnan) [inline places/##mayakrishnan-hometown.md:2]: 👍

Comment (fcortez1) [inline places/##mayakrishnan-hometown.md:2]: Can you provide best restaurants in Chennai?

Comment (ChrisH75) [inline places/##mayakrishnan-hometown.md:2]: @mayakrishnan Would love to visit :+1:

Comment (mayakrishnan) [inline places/##mayakrishnan-hometown.md:2]: Sure..Will see you there 👍

**Pull request merged**